code/_onclick/hud/hud.dm

```diff
@@ -81,6 +81,13 @@ GLOBAL_LIST_INIT(available_ui_styles, list(
 		plane_masters["[instance.plane]"] = instance
 		instance.backdrop(mymob)
 
+	wanted_lvl = new /obj/screen()

```

Comment:
Why is this always instanced? What if I'm not playing families? waste of resources.

Reply:
hud code is abysmal, it needs a rework but that's out of scope of this gamemode pr

Reply:
however i can make this new in the gamemode

Reply:
that's not an excuse to instance unused screen objects goof. just because code is shit doesn't mean stacking the shit pile higher is a good idea.

Reply:
I literally just said I was going to fix that. Are you even reading what I post, dude?

Reply:
you realize if you post in 2 comments and I reply to the first comment without refreshing I won't see it right